cscs-checks/prgenv/opencl_check.py

```diff
@@ -19,6 +19,7 @@ def __init__(self):
         self.valid_prog_environs = ['PrgEnv-cray', 'PrgEnv-pgi']
         self.modules = ['craype-accel-nvidia60']
         self.build_system = 'Make'
+        self.sourcesdir += os.path.join('/opencl')
         self.num_gpus_per_node = 1
         self.executable = 'vecAdd_opencl'
```

Comment:
If you move it to a specific directory, perhaps you don't need the `_opencl` suffix anymore.

Reply:
done